src/index.js

```diff
@@ -221,6 +225,8 @@ class ServerlessVpcPlugin {
     if (createDbSubnet) {
       if (numZones < 2) {
         this.serverless.cli.log('WARNING: less than 2 AZs; skipping subnet group provisioning');
+      } else if (subnetGroup.length > 0) {
```

Comment:
I think we can remove this check and always pass it into `buildSubnetGroups` even if its an empty array

Reply:
I got it, thanks